src/systems/physics/Physics.cc

```diff
@@ -663,10 +663,13 @@ void Physics::Configure(const Entity &_entity,
     return;
   }
 
-  auto classNames = pluginLoader.AllPlugins();
+  auto classNames = pluginLoader.PluginsImplementing<
```

Comment:
Can you comment on why this is necessary?

Reply:
When we were using `RTLD_GLOBAL`, we could find other plugins that have been loaded in the same process before. We're not using that anymore, so this is not required, but I think it's a nice extra filter just in case?

Reply:
I see. Thanks!

Reply:
I separated this into another PR just in case: https://github.com/ignitionrobotics/ign-gazebo/pull/965